client/src/components/books/categories/CategoriesList.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import TimeAgo from 'react-timeago';
+import Button from '../../form/Button';
+
+
+/* eslint-disable require-jsdoc, class-methods-use-this */
+class CategoriesList extends React.Component {
```

Comment:
If this is going to remain a presentational component, you don't need the class syntax. All you need do is to make it a stateless functional component that takes in props and returns an element.

```
export default ({ prop1, prop2 }) => {
  return (
    <div>{prop1, prop2}</div>
  )
}

Reply:
You can also destructure your props directly in the function definition rather than creating new variable to hold them.

Reply:
Thank you for the feedback, Femi